packages/runtime/src/api/fragments.ts

```diff
@@ -0,0 +1,5 @@
+import { DocumentNode } from 'graphql'
```

Comment:
Lifted up `fragments` and `types` from the `graphql` dir to `api` to draw a distinction between GraphQL's generated types and runtime API types, with the latter being an extension of the former (see below).